src/icon/colorIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export type ColorIconProps = Icon &
+  Readonly<{
+    borderColor?: string
```

Comment:
Also which colors are allowed? the whole palette? Should you type the colors?

Reply:
Color is sent by the backend (as explained in the FADR). We don't know what they'll send us, even if I don't think it changes that often.